tests/yam/validate/validate_installed_patterns.pm

```diff
@@ -0,0 +1,28 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Validate installed patterns
```

Comment:
Validate installed patterns against the list of additional patterns selected to be installed during installation.
Because we have also `tests/console/validate_installed_patterns.pm` which validate against patterns in the system.